cognite/client/data_classes/datapoints.py

```diff
@@ -271,16 +278,21 @@ def __str__(self):
             i["datapoints"] = utils._time.convert_time_attributes_to_datetime(i["datapoints"])
         return json.dumps(item, default=lambda x: x.__dict__, indent=4)
 
-    def to_pandas(self, column_names="externalId") -> "pandas.DataFrame":
+    def to_pandas(self, column_names: str = "externalId", include_aggregate_name: bool = True) -> "pandas.DataFrame":
```

Comment:
Should have a unit test for `include_aggregate_name` here aswell.

Reply:
done